packages/ui/VirtualList/UiVirtualListBase.js

```diff
@@ -270,7 +270,7 @@ const VirtualListBaseFactory = (type) => {
 		componentDidMount () {
 			if (!this.props.clientSize) {
 				this.calculateMetrics(this.props);
-				this.setState(this.getStatesAndUpdateBounds(this.props));
+				this.setState(this.getStatesAndUpdateBounds(this.props)); // eslint-disable-line react/no-did-mount-set-state
```

Comment:
I don't see any warnings related to this rule. 🤷

Reply:
I removed them from UiVirtualListBase and VirtualListBasic.